src/java.base/share/classes/sun/security/tools/keytool/Main.java

```diff
@@ -4580,10 +4579,10 @@ private CertificateExtensions createV3Extensions(
                             data = new byte[value.length() / 2 + 1];
                             int pos = 0;
                             for (char c: value.toCharArray()) {
-                                if (!hexFmt.isHexDigit(c)) {
+                                if (!HexFormat.isHexDigit(c)) {
```

Comment:
`hexFmt` created on line 4576 appears to be unused after this.

Reply:
The toHexDigit methods need access to the uppercase/lowercase distinction.
So there is no plan to change them.
I'll amend to include the above and re-check for uncaught uses.

Reply:
Hmm, I missed that unfortunate detail. It looks mis-matched to have `fromHex*` be static while `toHex*` variants aren't.